content/docs/reference/providers/pulumi/aws/s3/bucket.md

```diff
@@ -0,0 +1,1827 @@
+
+{{% /md %}}</td>
+        </tr>
+        <tr>
+            <td class="align-top">request<wbr>Payer</td>
+            <td class="align-top"><code>string</code></td>
+            <td class="align-top">{{% md %}}
+(Optional) Specifies who should bear the cost of Amazon S3 data transfer.
+Can be either `BucketOwner` or `Requester`. By default, the owner of the S3 bucket would incur
+the costs of any data transfer. See [Requester Pays Buckets](http://docs.aws.amazon.com/AmazonS3/latest/dev/RequesterPaysBuckets.html)
+developer guide for more information.
+
+{{% /md %}}</td>
+        </tr>
+        <tr>
+            <td class="align-top">server<wbr>Side<wbr>Encryption<wbr>Configuration</td>
+            <td class="align-top"><code><a href="#bucketserversideencryptionconfiguration">Bucket<wbr>Server<wbr>Side<wbr>Encryption<wbr>Configuration</a></code></td>
+            <td class="align-top">{{% md %}}
+(Optional) A configuration of [server-side encryption configuration](http://docs.aws.amazon.com/AmazonS3/latest/dev/bucket-encryption.html) (documented below)
+
+{{% /md %}}</td>
+        </tr>
+        <tr>
+            <td class="align-top">tags</td>
+            <td class="align-top"><code>Map&lt;<wbr>any<wbr>&gt;</code></td>
+            <td class="align-top">{{% md %}}
+(Optional) A mapping of tags that identifies subset of objects to which the rule applies.
+The rule applies only to objects having all the tags in its tagset.
+
+{{% /md %}}</td>
+        </tr>
+        <tr>
+            <td class="align-top">versioning</td>
+            <td class="align-top"><code><a href="#bucketversioning">Bucket<wbr>Versioning</a></code></td>
+            <td class="align-top">{{% md %}}
+(Optional) A state of [versioning](https://docs.aws.amazon.com/AmazonS3/latest/dev/Versioning.html) (documented below)
+
+{{% /md %}}</td>
+        </tr>
+        <tr>
+            <td class="align-top">website</td>
+            <td class="align-top"><code><a href="#bucketwebsite">Bucket<wbr>Website</a></code></td>
+            <td class="align-top">{{% md %}}
+(Optional) A website object (documented below).
+
+{{% /md %}}</td>
+        </tr>
+        <tr>
+            <td class="align-top">website<wbr>Domain</td>
+            <td class="align-top"><code>string</code></td>
+            <td class="align-top">{{% md %}}
+(Optional) The domain of the website endpoint, if the bucket is configured with a website. If not, this will be an empty string. This is used to create Route 53 alias records.
+
+{{% /md %}}</td>
+        </tr>
+        <tr>
+            <td class="align-top">website<wbr>Endpoint</td>
+            <td class="align-top"><code>string</code></td>
+            <td class="align-top">{{% md %}}
+(Optional) The website endpoint, if the bucket is configured with a website. If not, this will be an empty string.
+
+{{% /md %}}</td>
+        </tr>
+    </tbody>
+</table>
+
+## Bucket Output Properties
```

Comment:
This section would be so much more useful if it filtered out the properties that are also input properties. The docs aren't any different and if I'm explicitly looking for output properties, I probably only expect to see the outputs -- that is, the properties computed by the resource provider. Will make this way more streamlined and useful.